### PR TITLE
[Added TS] Update SiriusXM Presence

### PR DESCRIPTION
**Pull request by SnoweyFoxx**
Using Typescript

[Proof it's working with TS]
image
image
image
image

### CODE REVIEW

File: SiriusXM/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/"
+    }
+  }
```
Comment (Bas950) [inline SiriusXM/tsconfig.json:6]: https://docs.premid.app/en/dev/presence#filling-in-the-tsconfigjson-file

File: SiriusXM/presence.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
+var presence = new Presence({
+    clientId: "704186478410072065" //The client ID of the Application created at https://discordapp.com/developers/applications
+  });
```
Comment (Bas950) [inline SiriusXM/presence.ts:3]: remove comment

**Comment by Bas950** [comment]
this one isnt fixed: https://github.com/PreMiD/Presences/pull/1448#discussion_r417662124

**Comment by SnoweyFoxx** [comment]
Issues should be resolved.

**Pull request merged**